### ISSUE TITLE
Automatically choose first option in the completion menu

### ISSUE THREAD

**Issue by qazip** [hrsh7th/nvim-compe#367]
#### Is your feature request related to a problem? Please describe.
I am migrating from sublime-text to vim and I am trying to make it so completion and snippets work like in Sublime. Say I have a python file with 2 words:

```
for123
forabc
```
Now I type "for" and I want to enable the snippet for "for" (that comes from vim-vsnip). In sublime-text, I can just do "for+tab" and it expands the snippets perfectly. However, as it is, in vim, I have to do "for+down-arrow+tab". 

I would like to avoid having to press the down-arrow to choose the snippet. Essentially, I would like snippets to be 1st in the completion menu and I would like the first option to be automatically chosen so I can just do "for+tab" without having to choose that option.

Sorry for the wall of text! Is this something that can be accomplished with nvim-compe?

These are my settings for vsnip/nvim-compe:

```
imap <expr> <Tab> vsnip#jumpable(1) ? '<Plug>(vsnip-jump-next)' : compe#confirm("\<Tab>")
imap <expr> <S-Tab> vsnip#jumpable(-1) ? '<Plug>(vsnip-jump-prev)' : '<S-Tab>'
smap <expr> <Tab> vsnip#jumpable(1) ? '<Plug>(vsnip-jump-next)' : '<Tab>'
smap <expr> <S-Tab> vsnip#jumpable(-1) ? '<Plug>(vsnip-jump-prev)' : '<S-Tab>'
```
With also:
```
imap <expr> <C-j>   vsnip#expandable()  ? '<Plug>(vsnip-expand)'         : '<C-j>'
smap <expr> <C-j>   vsnip#expandable()  ? '<Plug>(vsnip-expand)'         : '<C-j>'
```

C-j works almost as I want, the only problem being that it is not tab! If I change C-j for tab, it stops working as I want...

To sum it up, what I want is this:
- If no option is selected, I want "tab" to expand the first snippet. How can I do that? Sorry, I am a noob in vim-speak..

**Comment by qazip** [comment]
Changing preselect to 'always' fixed my issue! F$$$ ya! Thank you!

**Comment by qazip** [comment]
Adding this to the lua config fixed my issue:
```

  source = {
    sort = { priority = 500; };
    tags = { priority = 500; };
    spell = { priority = 500; };
    path = { priority = 500; };
    buffer = { priority = 500; };
    calc = { priority = 500; };
    nvim_lsp = { priority = 500; };
    nvim_lua = { priority = 500; };
    vsnip = { priority = 1000; };

```
